layouts/partials/cover.html

```diff
@@ -0,0 +1,35 @@
+{{- $IsHome := .IsHome }}
+
+{{- with .cxt}} {{/* Apply proper context from dict */}}
+{{- if .Params.cover.image }}
+
+{{- $sizes := (slice "480" "720" "1080" "1500") }}
+
+{{- $src := cond (default true .Site.Params.responsiveImages) (.Page.Resources.GetMatch (printf "*%s*" (.Params.cover.image))) false }}
+{{- $fallback := cond (default false .Params.cover.relative) (path.Join .RelPermalink .Params.cover.image) (.Params.cover.image) -}}
+
+{{- $addLink := and (not $IsHome) (default true .Site.Params.linkFullImages) -}}
+{{- $responsiveImg := and $src (default false .Params.cover.relative) -}}
+
+<figure class="entry-cover">  
+{{- if $responsiveImg -}}
+    {{- if $addLink }}<a href="{{ $src.RelPermalink }}" target="_blank" title="Original image">{{ end }}
```

Comment:
We should add `| absURL` to every `.RelPermalink`.

Reply:
I'm not sure why you insist on absolute permalinks, but I've added them.